.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  build:
+    runs-on: windows-2019
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v1.7.2
+        with:
+          dotnet-version: 5.0.x
+      - name: Build
+        run: dotnet build src --configuration Release
+      - name: Publish artifacts
```

Comment:
Why are we uploading artifacts for both workflows?

Reply:
For CI, it's the only way to get at what was built. For Release, it's the only way to troubleshoot if what is deployed in the zip file is messed up in some way (though I could see killing that one). In both cases I figured 1 day is more than enough.